emmet-core/emmet/core/xrd.py

```diff
@@ -113,5 +119,6 @@ def from_target(
             edge=edge,
             min_two_theta=min_two_theta,
             max_two_theta=max_two_theta,
+            symprec=symprec,
```

Comment:
Great catch! I wonder what effect this bug has on our current XRD docs (if any).